README.rst

```diff
@@ -0,0 +1,105 @@
+Description |BuildStatus|_
```

Comment:
@leandro-lucarella-sociomantic, Should I look into setting up build and test on `travis-ci.org` or what you suggest? 
Any case I could make an issue for this.